packages/syft/src/syft/client/client.py

```diff
@@ -656,6 +656,49 @@ def me(self) -> Optional[Union[UserView, SyftError]]:
             return self.api.services.user.get_current_user()
         return None
 
+    @property
+    def registries(self) -> Any:
+        if self.api.has_service("image_registry"):
+            return self.api.services.image_registry
+        return None
+
+    @property
+    def images(self) -> Any:
+        if self.api.has_service("worker_image"):
+            return self.api.services.worker_image
+        return None
+
+    def build_image(
```

Comment:
so the reason why we go with `build_image` and not `push_image` is due to implementing the password prompt?

Reply:
sorry wrote the comment wrong earlier. fixed.

Reply:
@rasswanth-s

Reply:
crct, mainly for the prompts part, as we cannot get prompts by services

Reply:
I pushed a POC (untested) at #8363.

Reply:
we can take this up later once we tackle registry credentials in general?

Reply:
Should we take out `build_image` and `push_image` from this PR too? Can still do it currently with `self.api.services.worker_image.build/push`. As we're switching to `client.images.build/push` we should avoid exposing APIs that we mean to replace later.

Reply:
Why remove these? They are part of the UX in the PRD signed off by Irina.

Reply:
After discussion yesterday with Irina we agree that `client.images.build/push` is cleaner and more consistent with other services. The doc was updated.

Reply:
ah - then yes this will be removed